### DISCUSSION TITLE
Cypress: PageLoad Error Getting in Edge & Chrome Only while Bulk Execution

### DISCUSSION THREAD

**Discussion by pavanmpr** [cypress-io/cypress#28395] [Questions and Help]
in Cypress: I'm trying to run multiple test cases via  npx run with mutliple --spec , First case executing successfully then next case onwards getting pageload error but Page Loaded properly already...This issue happens only in Edge and Chrome not happening in FireFox....May I know why it happening?

**Comment by adnanerlansyah403** [comment]
It's the same issue that I got, but I also already made the thread for this and still don't get any response from the others. I hope this issue can keep up
